src/helpers/ast/is-literal.js

```diff
@@ -0,0 +1,4 @@
+// Literals, and `undefined`, are literals
+export default function isLiteral(t, node) {
+  return t.isLiteral(node) || (t.isIdentifier(node) && node.name === "undefined");
+}
```

Comment:
Have a different name in mind? iDOM isn't compatible with ES3 environments, so I figured it's safe to call it a literal.

It's needed so something like this doesn't eagerly evaluate or included unnecessary runtime helpers:

```jsx
function eager() {
  var a = <a href={undefined} />;
  return a;
};
// Before, that would be eagerly evaluated and placed into the dynamics
function eager() {
  var _undefined = undefined;
  var a = function() { return elementVoid('a', null, null, 'href', _undefined); };
  return a();
};
```